controllers/servicediscovery_controller.go

```diff
@@ -57,6 +57,8 @@ import (
 
 const (
 	clusterNotReadyRequeueTime = time.Minute * 2
+	// VSphereMachineControllerName is the name of the vSphere machine controller.
+	VSphereMachineControllerName = "vspheremachine-controller"
```

Comment:
Let's move this const inside `vspheremachine_controller.go`

Reply:
oops, mistakenly added.